src/main/java/com/ode22/catnews_origins/CatGuiController.java

```diff
@@ -91,6 +91,11 @@ public class CatGuiController implements Initializable {
     @FXML
     void onOpenTodaysFile(ActionEvent event) {
         System.out.println("Open pressed");
+        try {
+            fileHandler.openDailyFile();
+        } catch (IOException e) {
+            throw new RuntimeException(e);
```

Comment:
Rather then just throwing an error, we could notify the user with an error prompt that something went wrong while trying to open the file

Reply:
Good remark! Now an error opens an alert window to inform the user